File: C#-CarRacingProject/Domain/Inscriere.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProiectCSharpVersion.Domain
{
    class Inscriere : Entity<int>
    {
        public Participant Participant{get; set;}
        public Cursa Cursa{ get; set; }

        public Inscriere(Participant participant, Cursa cursa)
        {
            this.Participant = participant;
            this.Cursa = cursa;
        }

        public override string ToString()
        {
            return "Inscriere: " + Participant +"  "+ Cursa;
        }
    }
}

File: CarRacing_Client_Server_HibernatePersistence/CurseModel/src/main/java/curse/model/Inscriere.java
package curse.model;

public class Inscriere extends Entity<Integer>{
    private Participant participant;
    private Cursa cursa;

    public Inscriere(Participant p, Cursa c){
        participant=p;
        cursa=c;
    }
    public Inscriere(){
    }

    public Participant getParticipant() {
        return participant;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    public Cursa getCursa() {
        return cursa;
    }

    public void setCursa(Cursa cursa) {
        this.cursa = cursa;
    }

    @Override
    public String toString() {
        return "Inscriere{"+
                "Participant="+participant+
                "Cursa="+cursa+
                "}";
    }
}
